modules/couchbase/src/main/java/org/testcontainers/couchbase/UserDefinition.java

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
In general, we do not add the license headers to our files (yes, I know that there are some)

Could you please remove it?